packages/ui/components/AppCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { Info, ServiceInfoBadge } from './ServiceInfoBadge';
+
+interface AppCardProps {
+  isActive?: boolean;
+  name: string;
+  description: string;
+  badges: Info[];
+  isClient: boolean;
+  isServer: boolean;
+  className?: string;
+}
+
+export const AppCard = ({isActive = false, name, description, badges, className, isClient, isServer}:AppCardProps) => {
+  return (
+    <>
+      <div
+        className={`bg-gray-800 border-gray-800 rounded-lg w-[523px] border-2 ${className} ${
+          isActive ? ' border-pink-800/30 shadow-active' : ''
+        }`}
+      >
+        <div className="flex flex-col gap-2 px-5 py-3">
+          <h3 className="text-base font-medium text-gray-100">{name}</h3>
+          <div className='flex gap-1'>
+            { (isClient || isServer) && 
+            <div className='mr-2 flex gap-1'>
+              {isClient && <ServiceInfoBadge info='client'/>}
+              {isServer && <ServiceInfoBadge info='server'/>}
+            </div>
+            }
+            {badges.map((badge, index) => (<ServiceInfoBadge info={badge} key={index} />))}
```

Comment:
```suggestion
            {Array.from(new Set(badges))
              .filter(badge => validBadges.includes(badge))
              .map((badge, index) => (
                <ServiceInfoBadge info={badge} key={badge + index} aria-label={`Badge for ${badge}`} />))}
```
- I noticed that invalid badges leave a blank space (reserved for that invalid badge). 
What do you think about adding a check for this also? If a badge is not valid, we could simply ignore it instead of reserving blank space.

For this, we need to add an export of the list of valid badges from `ServiceInfoBadge`
```
export const validBadges: Info[] = ['http', 'kafka', 'websocket', 'amqp', 'mqtt', 'googlepubsub', 'ibmmq', 'nats', 'pulsar', 'redis', 'sns', 'sqs', 'solace', 'stomp', 'client', 'server'];
```

Reply:
we are already marking it's type as `Info[]` so it will give error in Typescript. so I am more in favour of letting typescript give the error.

Reply:
Sorry, I didn't get this, could you please explain that further.

Reply:
@princerajpoot20 I mean Typescript doesn't allow invalid badges:

<img width="641" alt="Screenshot 2023-11-29 at 16 33 31" src="https://github.com/asyncapi/studio/assets/32505158/b6ba8a40-3d30-4f19-b0da-b5e7d3f28a4d">

Reply:
@KhudaDad414 Okay, got it. Makes sense. 👍